src/app/legacy/containers/Navigation/index.jsx

```diff
@@ -38,8 +38,8 @@ const renderListItems = (
         currentPageText={currentPage}
         service={service}
         dir={dir}
-        clickTrackerHandler={clickTrackerHandler}
-        viewTracker={viewTracker}
+        {...clickTrackerHandler}
+        viewRef={viewTracker}
```

Comment:
All references of viewRef have now been renamed to `viewTracker`
```suggestion
        viewTracker={viewTracker}
```